main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/cloudflare/cloudflare-go"
+	csbouncer "github.com/crowdsecurity/go-cs-bouncer"
+)
+
+func deleteExistingCrowdSecIpList(ctx context.Context, api *cloudflare.API) {
+	ipLists, _ := api.ListIPLists(ctx)
+	id, err := getCrowdSecIPListId(ipLists)
+	if err != nil {
+		return
```

Comment:
can you please return err here and check the err when calling the function to fatal